src/components/BoardGame.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div class="dashboard">
+    <h3>{{board.title}}</h3>
+    <div class="game-board">
+      <table>
+        <tr v-for="(row, index) in rows" :key="index">
+          <td v-for="(col, index) in cols" :key="index">
+            .
+          </td>
+        </tr>
+      </table>
+    </div>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'boardgame',
+    props: ['board'],
+    computed: {
+      rows: function() {
```

Comment:
We should not use arrow function as mentioned in:
https://vuejs.org/v2/guide/instance.html#Data-and-Methods
=======================================
Don’t use arrow functions on an options property or callback, such as created: () => console.log(this.a) or vm.$watch('a', newValue => this.myMethod()). Since arrow functions are bound to the parent context, this will not be the Vue instance as you’d expect, often resulting in errors such as Uncaught TypeError: Cannot read property of undefined or Uncaught TypeError: this.myMethod is not a function.
=======================================

Reply:
agree, but I said "method shorthand" syntax (notice there is no arrow):
```javascript
someMethod() {
  . . . 
}
```
this is different than arrow functions:
```javascript
someFunction () => {
   . . . 
}
```